TestModels/SimpleTypes/SimpleEnum/runtimes/rust/src/operation.rs

```diff
@@ -0,0 +1,9 @@
+// Code generated by software.amazon.smithy.rust.codegen.smithy-rs. DO NOT EDIT.
+
+/// Types for the `GetEnum` operation.
+pub mod get_enum;
+
+/// Types for the `GetEnumKnownValue` operation.
```

Comment:
This is a little off, there are three different operations.

Also the operations are named `GetEnumFirstKnownValueTest` etc., so we're missing the `_test` in general.

Reply:
It's quite possible other test models have this wrong too.